src/owncloud/autoconfig.php

```diff
@@ -3,23 +3,26 @@
 $snap_name = getenv('SNAP_NAME');
 $snap_developer = getenv('SNAP_DEVELOPER');
 if ($snap_developer == '') {
-        $snap_developer = getenv('SNAP_ORIGIN');
+	$snap_developer = getenv('SNAP_ORIGIN');
 }
 
 $snap_fullname = $snap_name . '.' . $snap_developer;
 
-$data_path = '/var/lib/snaps/'.$snap_fullname.'/current';
+$data_path = '/var/lib/snaps/' . $snap_fullname . '/current';
 
-$AUTOCONFIG = array(
-'directory' => $data_path.'/owncloud/data',
+$writable = getenv('SNAP_DATA');
+$owncloud_password = trim(file_get_contents($writable . '/mysql/owncloud_password'));
```

Comment:
Ah, it's clearer now, so I don't see a problem with having to wait for the password to be defined and saved to file in that `setup_owncloud` script. We have to wait for the DB to be set up anyway.

 >The file isn't actually destroyed

I was talking about autoconfig.php

>but the variable is definitely not necessary anymore.

It's already remove in what I'll push later